.pre-commit-config.yaml

```diff
@@ -0,0 +1,30 @@
+exclude: '^docs/conf.py'
+
+repos:
+- repo: git://github.com/pre-commit/pre-commit-hooks
+  rev: v2.2.3
+  hooks:
+  - id: trailing-whitespace
+  - id: check-added-large-files
+  - id: check-ast
+  - id: check-json
+  - id: check-merge-conflict
+  - id: check-xml
+  - id: check-yaml
+  - id: debug-statements
+  - id: end-of-file-fixer
+  - id: requirements-txt-fixer
+  - id: mixed-line-ending
+    args: ['--fix=no']
+  - id: flake8
+    args: ['--max-line-length=88']  # default of Black
+
+- repo: https://github.com/pre-commit/mirrors-isort
+  rev: v4.3.4
+  hooks:
+  - id: isort
+
+-   repo: https://github.com/ambv/black
```

Comment:
It seems like this is indented more than the previous hooks. Is this on purpose?

Reply:
Nope. I'll fix that. Doesn't seem to break things though.